x-pack/plugins/observability_ai_assistant/server/plugin.ts

```diff
@@ -91,21 +85,12 @@ export class ObservabilityAIAssistantPlugin
           ui: ['show'],
         },
         read: {
-          app: [OBSERVABILITY_AI_ASSISTANT_FEATURE_ID, 'kibana'],
-          api: [OBSERVABILITY_AI_ASSISTANT_FEATURE_ID, 'ai_assistant'],
-          catalogue: [OBSERVABILITY_AI_ASSISTANT_FEATURE_ID],
+          disabled: true,
```

Comment:
__question:__ I assume we never released this feature, right? Otherwise it'd be a breaking change.

Reply:
Correct